src/openms/source/FORMAT/MSPGenericFile.cpp

```diff
@@ -130,22 +130,22 @@ namespace OpenMS
       // Specific case of NIST's exported msp
       else if (boost::regex_search(line, m, re_cas_nist))
       {
-        // LOG_DEBUG << "CAS#: " << m[1] << "; NIST#: " << m[2] << "\n";
+        // OPENMS_LOG_DEBUG << "CAS#: " << m[1] << "; NIST#: " << m[2] << "\n";
         spectrum.setMetaValue(String("CAS#"), String(m[1]));
         spectrum.setMetaValue(String("NIST#"), String(m[2]));
       }
       // Other metadata
       else if (boost::regex_search(line, m, re_metadatum))
       {
-        // LOG_DEBUG << m[1] << m[2] << "\n";
+        // OPENMS_LOG_DEBUG << m[1] << m[2] << "\n";
         spectrum.setMetaValue(String(m[1]), String(m[2]));
       }
     }
     // To make sure a spectrum is added even if no empty line is present before EOF
     addSpectrumToLibrary(spectrum, library);
-    ifs.close();
-    LOG_INFO << "Loading spectra from .msp file completed." << std::endl;
-    std::cout << "PARSE TIME: " << ((std::clock() - start) / (double)CLOCKS_PER_SEC) << std::endl;
+    OPENMS_LOG_INFO << "Loading spectra from .msp file completed." << std::endl;
+    // DEBUG
+    // std::cout << "PARSE TIME: " << ((std::clock() - start) / (double)CLOCKS_PER_SEC) << std::endl;
```

Comment:
looks like this leads to an error because you are not using the variable `start` any more.  Also, why are you not closing `ifs` any more?
```suggestion
    OPENMS_LOG_INFO << "PARSE TIME: " << ((std::clock() - start) / (double)CLOCKS_PER_SEC) << std::endl;
```

Reply:
`.close()` is called automatically by the object's destructor, hence I preferred to remove the line because unnecessary.

I will comment out the unused variable's declaration.